datasets/pk/proscribed_persons/crawler.py

```diff
@@ -0,0 +1,33 @@
+from zavod import Context
+from zavod import helpers as h
+
+
+def crawl_person(context: Context, row: dict):
+    person_name = row.pop("name")
+    father_name = row.pop("fatherName")
+    cnic = row.pop("cnic")
+    province = row.pop("province")
+    district = row.pop("district")
+
+    entity = context.make("Person")
+    entity.id = context.make_slug(person_name, prefix="pk-proscribed")
+    entity.add("idNumber", cnic)
+    entity.add("name", person_name)
+    entity.add("fatherName", father_name)
+    entity.add("topics", "crime.terror")
+
+    address_entity = h.make_address(
+        context, state=province, region=district, country_code="pk"
```

Comment:
can we assume pakistan? perhaps it's safest to skip the country_code unless there's something suggesting it will always be places in Pakistan

Reply:
So one of the provinces I happened to google turns out to be a region shared by Iran, Afghanistan and Pakistan so I think sticking the pk country code on it isn't the right thing.

I asked Friedrich about this address and the merged id issue and (a) whether it's expected that such a limited address is expected to generate an emtpy `full`, and (b) whether this should be an address entity or just a string in the address property.

a. the decision was a bug, I reported it at https://github.com/pudo/addressformatting/issues/1 because we won't fix it right now
b. it's so limited so just combine `{region}, {state}` and stick it into the entity address property

Reply:
Ok I understand.
I'll stick the `region}, {state}` for now.